plugins/loader/awaitloader/AwaitLoaderCallback.d.ts

```diff
@@ -4,20 +4,20 @@ declare namespace AwaitLoaderCallback {
     type RunTaskCallbackType = (
         successCallback: Function,
         failureCallback: Function
-    ) => void;
+    ) => Promise<void> | void;
```

Comment:
We use AwaitLoader in our project for wrapping requests to API, and we use async/await for this purpose. And it works 🤔 

So, I think it's okay to add Promise support here too, because it should not break anything.